fedora30-test-container/Dockerfile

```diff
@@ -25,8 +25,8 @@ RUN dnf clean all && \
     libffi-devel \
     make \
     mariadb-server \
-    openssh-clients \
-    openssh-server \
+    openssh-clients-7.9p1 \
```

Comment:
Should we be using the older version, is there a way to convert the OpenSSH key to a PEM key manually?

Reply:
I'm using the older version just until the private key is generated. Then it gets updated to the latest version.